build/.azure-pipelines.TemplateValidation.yml

```diff
@@ -79,17 +79,15 @@ jobs:
         templateArgs: '-preset=recommended -auth=msal -markup csharp'
       MsalAuthMvvmCSharp:
         templateArgs: '-preset=recommended -presentation mvvm -auth=msal -markup csharp'
-      # NOTE: Temporarily disabling so we can merge #585
-      # MauiBlank:
-      #  templateArgs: '-preset=blank -maui'
-      # MauiRecommended:
-      #  templateArgs: '-preset=recommended -maui'
+      MauiBlank:
+       templateArgs: '-preset=blank -maui'
+      MauiRecommended:
+       templateArgs: '-preset=recommended -maui'
       ThemeService:
        templateArgs: '-theme-service'
-      # NOTE: Temporarily disabling so we can merge #585
       # https://github.com/unoplatform/uno.templates/issues/396
```

Comment:
Do we need to keep this as #396 is closed?

Reply:
Yes these tests are there to ensure we do not have regressions